tests/unit/terra.test.js

```diff
@@ -0,0 +1,188 @@
+import { Client } from '@liquality/client'
+import { ChainNetworks } from '@/utils/networks'
+import { TerraNetworks } from '@liquality/terra-networks'
+import { TerraRpcProvider } from '@liquality/terra-rpc-provider'
+import { TerraWalletProvider } from '@liquality/terra-wallet-provider'
+import { TerraSwapProvider } from '@liquality/terra-swap-provider'
+import { TerraSwapFindProvider } from '@liquality/terra-swap-find-provider'
+
+test('Terra Network TesnetConfig', () => {
+  expect(TerraNetworks.terra_testnet).toEqual({
+    name: 'testnet',
+    networkId: 'testnet',
+    nodeUrl: 'https://bombay-lcd.terra.dev',
+    helperUrl: 'https://bombay-fcd.terra.dev/v1',
+    gasPricesUrl: 'https://bombay-fcd.terra.dev/v1/txs/gas_prices',
+    coinType: '397',
+    isTestnet: true,
+    chainID: 'bombay-10',
+    asset: 'uluna',
+    codeId: 7685
+  })
+})
+
+test('Terra Network Mainnet', () => {
+  expect(TerraNetworks.terra_mainnet).toEqual({
+    name: 'mainnet',
+    networkId: 'mainnet',
+    nodeUrl: 'https://lcd.terra.dev',
+    helperUrl: 'https://fcd.terra.dev/v1',
+    gasPricesUrl: 'https://fcd.terra.dev/v1/txs/gas_prices',
+    coinType: '397',
+    isTestnet: false,
+    chainID: 'columbus-5',
+    asset: 'luna',
+    codeId: 6431
+  })
+})
+
+test('Create Client for testnet and attach Terra RpcProvider successfully', async () => {
```

Comment:
Tests below here all seem like they should belong in CAL...
You can maybe add some further test cases to derivation path tests?
Or test some logic in the permission screen

Reply:
I'll remove this file and do these tests in CAL looks like more suitable and more correct place than here.